README.md

```diff
@@ -83,6 +89,13 @@ We are providing example apps for Swift and Objective-C. These apps demonstrate
 - iOS 8.0+
 - Xcode 8.0+
 
+**Note:**
+In order to have better analysis results, although is not a requirement, It's highly recommended to have devices with 8MP camera and flash. These devices would be:
```

Comment:
In order to have better analysis results, ~~although is not a requirement, It's~~ it is highly recommended to ~~have~~ enable only devices with 8MP camera and flash.